src/main/java/net/syscon/elite/repository/jpa/model/CourtEvent.java

```diff
@@ -31,7 +31,7 @@
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-@EqualsAndHashCode(of = "id", callSuper = false)
+@EqualsAndHashCode(callSuper = false)
```

Comment:
Removing ID is fine I guess - I assume so you have test equality in your tests but collections with a full set of attributes in equals could cause extra loading of data - not a problem here I suspect

Reply:
Yeah.  I am really on the fence with this one.  Perhaps we can take this offline.